### PR TITLE
Update top level documents for compliance

### PR DESCRIPTION
**Pull request by bobbrow**
Updating the readme, security, code of conduct, and contributing guides.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,3 +33,18 @@ const localize: nls.LocalizeFunc = nls.loadMessageBundle();
...
+and actually do, grant us the rights to use your contribution. For details, visit
+https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need
+to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the
+instructions provided by the bot. You will only need to do this once across all repositories using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/)
+or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
```
Comment (sean-mcmanus) [inline CONTRIBUTING.md:50]: Should this line add ` Employees can reach out at [aka.ms/opensource/moderation-support](https://aka.ms/opensource/moderation-support)`?

**Review by Colengms — APPROVED** [review]

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
I made a comment.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bobbrow) [reply CONTRIBUTING.md:50]: That line is already in the CODE_OF_CONDUCT.md and the guidance did not instruct me to put it here.

**Comment by bobbrow** [comment]
> I made a comment.

I responded to your comment.

**Review by sean-mcmanus — APPROVED** [review]

**Review by Colengms — APPROVED** [review]

**Pull request merged**